scripts/common.py

```diff
@@ -940,15 +940,9 @@ def managed_firebase_auth_emulator():
 
     emulator_args = [
         FIREBASE_PATH, 'emulators:start', '--only', 'auth',
-        '--project', feconf.OPPIA_PROJECT_ID
+        '--project', feconf.OPPIA_PROJECT_ID,
     ]
     with contextlib2.ExitStack() as stack:
-        # These two environment values allow the Firebase SDKs to acknowledge
```

Comment:
Can you explain why this changed? (as a reply to this review comment, not in the code)

Reply:
Because from local testing, I've found that we need to set these in `app.yaml` to get the Firebase SDK to acknowledge them.